module_6/tusk/index.js

```diff
@@ -0,0 +1,40 @@
+"use strict"
+
+// tusk #1
+const randomArr = [1, 2, "Ivan", "banana", "Kiev", "Table", 3, 4];
+
+function arrayCopy () {
+    console.log(randomArr.slice());
+}
+
+// tusk #2
+const arrTusk2 = ["The", 'quick', 'brown', 'fox', 'jumps', 'over', 'the', 'lazy', 'dog']
+
+function arrayJoin(conditionArr) {    
+    console.log(conditionArr.join(' '));
```

Comment:
```suggestion
    return conditionArr.join(delimiter);
```

console.log(arrTusk2, ' ');